### PR TITLE
fix: Add HSTS and RP headers to the base image, since we no longer use seckit.

### PR DESCRIPTION
**Pull request by cafuego**
This will make Beaglesecurity happier.

Refs: OPS-8746

### CODE REVIEW

File: php/php-k8s-v7/etc/nginx/nginx.conf [hunk 1]
```diff
@@ -119,11 +119,17 @@ http {
...
       ""        "1; mode=block";
    }
-   add_header X-XSS-Protection $xssprotection;
+  add_header X-XSS-Protection $xssprotection;
 
   ## Block MIME type sniffing on IE.
   add_header X-Content-Options nosniff;
 
+  ## Enable HSTS.
+  add_header Strict-tRansport-Security "max-age=31536000; includeSubDomains; preload; always;";
```
Comment (lazysoundsystem) [inline php/php-k8s-v7/etc/nginx/nginx.conf]: "Transport" in this and the other versions.

**Review by lazysoundsystem — DISMISSED** [review]
Apart from the capitalization, looks good. I think.

File: php/php-k8s-v7/etc/nginx/nginx.conf (hunk 1 above)
Comment (cafuego) [reply php/php-k8s-v7/etc/nginx/nginx.conf]: 🤦‍♂️

**Comment by berliner** [comment]
Is it an issue if seckit is still used? Like in FTS Public and Hum Insight?

**Comment by cafuego** [comment]
I hope not; we'll see with Sesame (which also still uses seckit)

**Comment by cafuego** [comment]
Offhand I suspect a duplicate HSTS header would be not ideal. We might need to turn them off in seckit.

**Comment by berliner** [comment]
Ok, ping me if there is any action required on my part.

**Comment by cafuego** [comment]
As per https://github.com/UN-OCHA/tools-ansible/pull/978

**Review by teodorescuserban — DISMISSED** [review]

**Comment by cafuego** [comment]
The updated config makes nginx use the backend-provided header values if present, or add the defined ones if they are absent. Working nicely on Sesame. This means no overrides are required for settings.php for sites that use SecKit.

**Review by teodorescuserban — APPROVED** [review]
nice one!

**Pull request merged**